### PR TITLE
Undo change intorduce by #31

### PR DESCRIPTION
**Pull request by maxime-rainville**
#31 shouldn't have been merged in the 1.2 branch because it breaks cwp-recipe-search 2.4.

### CODE REVIEW

File: src/CwpSearchResult.php [hunk 1]
```diff
@@ -62,15 +61,15 @@ class CwpSearchResult extends ViewableData
     public function __construct($terms = '', ArrayData $results = null)
     {
         $this->query = $terms;
-        $searchableService = SearchableService::singleton();
```
Comment (maxime-rainville) [inline src/CwpSearchResult.php:65]: I'm presuming this API got introduced in Fulltext 3.7?

Comment (emteknetnz) [reply src/CwpSearchResult.php:65]: That's correct

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +1,23 @@
...
-dist: trusty
+dist: xenial
+
+services:
+  - mysql
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION=1.x-dev
+    - COMPOSER_ROOT_VERSION=1.2.x-dev
```
Comment (emteknetnz) [inline .travis.yml:14]: Should probably not revert the travis file changes

Comment (maxime-rainville) [reply .travis.yml:14]: This branch is meant to target the 1.2.x release line. I'll switch it back to 1.x-dev on the merge up.

File: .travis.yml [hunk 3]
```diff
@@ -1,20 +1,23 @@
 language: php
 
-dist: trusty
+dist: xenial
+
+services:
+  - mysql
 
 env:
```
Comment (emteknetnz) [inline .travis.yml:12]:
```suggestion
cache:
  directories:
    - $HOME/.composer/cache/files

env:
```

**Review by emteknetnz — APPROVED** [review]
Squash merge on green

**Pull request merged**